v1.4.0: Additional alerting methods, ping overrides, DSM 7.1 support

beornlake opened the pull request:
- Adds configurable options to give hardware alerts (beeps and lights), as well as an indicator in the filesystem when a VPN loses connection.

peledies reviewed (approved):
This is a great idea!

I have tested this and it works as intended on my `DS214play`

beornlake commented:
- Added support for OpenVPN connections in DSM 7.1 (fixes #16).
- Removed support for PPTP VPNs.
- Added override options for `domain_ping` and `gateway_ping` link detection methods. (see https://github.com/ianharrier/synology-scripts/issues/13#issuecomment-952824266)
- Cleaned up document layout and formatting; reorganized some sections.
- Added instructions comment block to the top of the document.
- Bumped to version 1.4.0 (arbitrarily).

beornlake commented on reconnect-vpn.sh:
```diff
@@ -1,134 +1,231 @@
 #
-#       AUTHOR:  Ian Harrier
-#      VERSION:  1.2.1
+#       AUTHOR:  Ian Harrier, Michael Lake
```
```suggestion
#       AUTHOR:  Ian Harrier, Michael Lake, Deac Karns
```

ianharrier commented:
Thank you for contributing! I'd be interested in merging this if you could rebase on the latest version. I've already merged in some of the other features also included here. Let's keep this pull request focused on just adding the new alerting features. Thanks!
